doc/user/config.toml

```diff
@@ -237,4 +237,4 @@ include = "docs/**"
 
 # Avoid deleting the LTS and self-managed docs which are deployed from the
 # lts-docs and self-managed-docs branches.
-exclude = "docs/{lts,self-managed}/**"
+exclude = "docs/{lts,self-managed/v25.1}/**"
```

Comment:
I'll admit, not sure if this is the right pattern.

Reply:
Worst case scenario, can just exclude lts docs.
And then after the redirects are deployed, just republish from the self-managed branch.